src/main/AuthenticationResponse.ts

```diff
@@ -70,50 +71,25 @@ export default class AuthenticationResponse {
     verifiedJwt: VerifiedAuthenticationRequestWithJWT,
     responseOpts: AuthenticationResponseOpts
   ): Promise<AuthenticationResponseWithJWT> {
+    const idToken = await createSIOPIDToken(verifiedJwt, responseOpts);
```

Comment:
the naming is really confusing here. this `idToken` is the unsigned `id_token` then you will pass it to method `signDidJwtPayload` to get a signed version of it (which we previously called jwt) and again you're calling it id_token. suggest to change the names to a more clarifying version

Reply:
I don't think so. This is a pattern in the library, snake case for payloads or objects sent over the wire and camel case for objects that are locally used, like opts.  It's pretty straight forward, the `IdToken` object is created, signed, which obviously makes it a string because it is JWT and assigned to the request payload. It's the only place where you will see it, it's inside a single method and contained within 3 lines. It makes no sense to call it jwt in the payload, because there is no such property in the specs.